supporting_scripts/analysis-of-endpoint-connections/src/main/java/analysisOfEndpointConnections/AnalysisOfEndpointConnections.java

```diff
@@ -0,0 +1,75 @@
+package analysisOfEndpointConnections;
+
+import java.io.File;
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.List;
+import java.util.Optional;
+
+import com.thoughtworks.qdox.JavaProjectBuilder;
+import com.thoughtworks.qdox.model.JavaAnnotation;
+import com.thoughtworks.qdox.model.JavaClass;
+import com.thoughtworks.qdox.model.JavaMethod;
+
+public class AnalysisOfEndpointConnections {
+
+    /**
+     * This is the entry point of the analysis of server sided endpoints.
+     *
+     * @param args List of files that should be analyzed regarding endpoints.
+     */
+    public static void main(String[] args) {
+        if (args.length == 0) {
+            System.out.println("No files to analyze.");
+            return;
+        }
+        String[] filePaths = args[0].split("\n");
+        String[] serverFiles = Arrays.stream(filePaths).map(filePath -> "../../" + filePath).filter(filePath -> new File(filePath).exists() && filePath.endsWith(".java"))
```

Comment:
is this formatting intended?

Reply:
Yes. As the Action is now run using native gradle features, it is now located at a different place in the repository structure. The working directory now is: `/home/runner/work/Artemis/Artemis/supporting_scripts/analysis-of-endpoint-connections`.

To be able to check for the existence of modified files (to make sure they were not deleted, this would throw exceptions), we need to check in the Repository root. This is done by reformatting the file paths.